css/style.css

```diff
@@ -92,3 +92,313 @@ nav {
 ul li {
   list-style-type: none;
 }
+
+.action-button {
+  display: flex;
+  justify-content: center;
+  margin-top: 64px;
+  margin-bottom: 22px;
+}
+
+.recent-work header {
+  display: grid;
+  justify-content: center;
+}
```

Comment:
- [ ] I would strongly recommend you use `display: grid ` on the work section container and not on the inner element to layout the content of the work section. This gives you control to layout your content nicely and to make your website  responsive 😄.
e.g
```
recent-work  {
  display: grid;
  _other atttributes_
}
```

Reply:
Fixed